docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -263,11 +263,17 @@ spec:
   storageClassName: ''
   tenants:
     mode: openshift-logging
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 9
+        ingestionRate: 5
 ----
 
 A dictionary holding the `.spec` for the LokiStack resource.
 
```

Comment:
Side-note: please verify that loki configuration is in actual SI MB/s. If it is, the formula should divide by 1000 to get to GB/s instead of 1024.

```suggestion

The component configures fluentd as the default log forwarder.
The default chunk size limit in fluentd is 8 MB which the burst size limit of 9 MB accounts for.

The ingestion rate defines the MB/s limit for a tenant.
OpenShift Logging uses the following three tenants:

* application
* audit
* infrastructure

The max allowed volume for a tenant per day can be calculated with the following formula:

[stem]
++++
"volumePerDay"_"tenant" ("in GB/s") = ("ingestionRate"_"tenant" ("in MB/s") * 60 * 60 * 24) / 1000
++++

The default of 5 MB/s allows up to ~420 GB of logs per day for a tenant.
```

As far as I can tell, the formula should be rendered nicely on hub.syn.tools, if it's not please let me know.

Screenshot from local preview:  

![logging-docs-formula](https://user-images.githubusercontent.com/393160/220571537-cdbc9ee7-f738-413e-8d68-c16db1f693f5.png)

Reply:
It's based on the power of 2: https://github.dev/grafana/loki/blob/main/pkg/validation/limits.go#L47

Updated with the correct units.